src/components/ebay-button/browser.json

```diff
@@ -2,7 +2,10 @@
     "dependencies": [
         {
             "if-not-flag": "ebayui-no-skin",
-            "path": "@ebay/skin/button"
+            "dependencies": [
+                "@ebay/skin/actionable",
+                "@ebay/skin/button"
+            ]
         },
         "require: marko-widgets",
```

Comment:
Hi Sean,
Do we need to declare a dependency on ebay-badge here? Or is it unnecessary?

Reply:
Great callout. I believe I do. Will update.